src/pages/GameWordPage/GameWordPage.tsx

```diff
@@ -1,5 +1,87 @@
+import { ReactNode } from 'react';
+import IngameHeader from '@/common/Ingame/IngameHeader';
+import Input from '@/common/Input/Input';
+import { wordDummy, wordRankDummy } from './wordDummy';
+
+const EMPTY_WORD = 20;
+const positions = ['center', 'left', 'right'];
+const WordCell = ({ children }: { children: ReactNode }) => {
+  const random = positions[Math.floor(Math.random() * positions.length)];
+  return (
+    <span
+      className={`p-1 text-${random} ${Math.round(Math.random()) && 'mt-auto'} ${Math.round(Math.random()) && 'mb-auto'}`}>
+      {children}
+    </span>
+  );
+};
+// 총 120개의 cell. 서버로부터 받을 단어 100개 + 랜덤20개(EMPTY_WORD)로 구성해야함
+function shuffle(array: string[]) {
+  return array.sort(() => Math.random() - 0.5);
+}
```

Comment:
단어들을 섞는 함수를 잘 만드셨네요!! 😊
혹시 이 부분은 arrow function으로 바꾸면 통일성이 더 좋아질 것 같은데 이렇게 바꿔보시는 건 어떠실까용?!
```suggestion
const shuffle = (array: string[]) => {
  return array.sort(() => Math.random() - 0.5);
}
```

Reply:
오오,,컨벤션 주의하겠습니다 🤒